delta/sdk-testkit/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/OwnerPermissionsDummy.scala

```diff
@@ -0,0 +1,55 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.testkit
+
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError.ScopeInitializationFailed
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.{Acl, AclAddress}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.{Caller, ServiceAccount}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.organizations.Organization
+import ch.epfl.bluebrain.nexus.delta.sdk.model.permissions.Permission
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.Project
+import ch.epfl.bluebrain.nexus.delta.sdk.{Acls, ScopeInitialization}
+import monix.bio.IO
+
+/**
+  * Apply the owner permissions at org/project creation
+  *
+  * @param acls               an instance of the acl module
+  * @param ownerPermissions   the owner permissions to be present at project creation
+  * @param serviceAccount     the service account to apply owner permissions when needed
+  */
+class OwnerPermissionsDummy(acls: Acls, ownerPermissions: Set[Permission], serviceAccount: ServiceAccount)
```

Comment:
As it takes an acl (which can be a dummy), we should not need one for applying owner permissions itself no ?

Reply:
I'm not sure I understand your comment...

Reply:
`OwnerPermissionsScopeInitialization` and `OwnerPermissionsDummy` have the same dependencies, we should not need two implementations

Reply:
it's a module dependency problem... we need two implementations in order to maintain the dummy acl behaviour without bringing more deps on the sdkTestkit.